libr/anal/p/anal_arm_cs.c

```diff
@@ -2628,6 +2628,8 @@ jmp $$ + 4 + ( [delta] * 2 )
 		op->cycles = 1;
 		break;
 	case ARM_INS_POP:
+		op->stackop = R_ANAL_STACK_INC;
+		op->stackptr = -4LL * insn->detail->arm.op_count;
```

Comment:
add `// fallthrough` comment at the end of the block, or static code analysis tools may complain about possibly unvoluntary missing break